requirements-conda.txt

```diff
@@ -23,13 +23,6 @@ vphaser2=2.0
 biopython=1.68
 matplotlib=1.5.3
 future>=0.15.2
-PyYAML=3.11
 pysam=0.9.1
-pycodestyle
-mock==2.0.0
-six<2
-pytest=3.0.5
```

Comment:
That's true. I'll move these test-related packages back to the conda requirements.